spec/views/featured_attachments/index.html.erb_spec.rb

```diff
@@ -4,24 +4,31 @@
     edition = create(:edition,
                      file_attachment_revisions: [file_attachment_revision])
     assign(:edition, edition)
-
-    expect(render).to have_content(file_attachment_revision.title)
+    render
+    expect(rendered).to have_content(file_attachment_revision.title)
   end
 
   it "shows a message when there aren't any attachments" do
     assign(:edition, create(:edition))
-
-    expect(render).to have_content(I18n.t!("featured_attachments.index.no_attachments"))
+    render
+    expect(rendered).to have_content(I18n.t!("featured_attachments.index.no_attachments"))
   end
 
-  it "only shows a reorder action for multiple attachments" do
-    edition = create(:edition)
-    edition.file_attachment_revisions << build(:file_attachment_revision)
-
+  it "shows a reorder action for a multiple attachments" do
```

Comment:
```suggestion
  it "shows a reorder action for multiple attachments" do
```

Reply:
I kind of preferred having this as one test, as we are really testing "one thing".

Reply:
I'm not massively fussed either way but going down a single test route means we have to treat the rendered method different as it [appends to it with each render](https://github.com/rails/rails/blob/4d0947207a54db89b646b6277b9f8c8f6dcf53d6/actionview/lib/action_view/test_case.rb#L122) (which I just learnt while trying to move this back) and thus move the expectations back to being `expect(render).to have_content()`. 

I'm not particularly opposed to using render in that way it's just a tricky balance of consistency across the view tests.

Reply:
Oh, yuk. That's unfortunate - let's keep it like this!